### PR TITLE
Dev 7277 refactor ambassador objects

### PR DESCRIPTION
**Pull request by jgonzalezspoton**
### What
Refactoring Ambassador templates

### Why
cleanliness and share values across ambassador objects, making this easier to expand upon in the future if needed

### Changes
- move tld used by various ambassador objects to fall under ambassador.tld
- mappings, filters, filter policies templates moved to fall under ambassador

### Tests:

```
 √ . [spoton-gbl-identity] (HOST) monochart ⨠ k get mappings
NAME      SOURCE HOST                                  SOURCE PREFIX   DEST SERVICE                                  STATE   REASON
  apigw     apigw.express-nonprod-c1.spoton.sh           /               monochart-testing.default.svc.cluster.local
app       nginx-testing.express-nonprod-c1.spoton.sh   /               monochart-testing.default.svc.cluster.local
default   monochart.express-nonprod-c1.spoton.sh       /               monochart-testing.default.svc.cluster.local
grpc      nginx-grpc.express-nonprod-c1.spoton.sh      /               monochart-testing.default.svc.cluster.local
```

```
 √ . [spoton-gbl-identity] (HOST) monochart ⨠ curl https://apigw.express-nonprod-c1.spoton.sh/
Hello, this is the default route on port 80! ⧉  SpotOn Inc (uw2-express-nonprod-c1:N/A)+
 √ . [spoton-gbl-identity] (HOST) monochart ⨠ curl https://nginx-testing.express-nonprod-c1.spoton.sh/
Hello, this is the default route on port 80! ⧉  SpotOn Inc (uw2-express-nonprod-c1:N/A)+
 √ . [spoton-gbl-identity] (HOST) monochart ⨠ curl https://monochart.express-nonprod-c1.spoton.sh/
Hello, this is the default route on port 80! ⧉  SpotOn Inc (uw2-express-nonprod-c1:N/A)+
```

### CODE REVIEW

File: monochart/nginx-config-patch-monochart-testing.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+spec:
```
Comment (dalefrancum) [inline monochart/nginx-config-patch-monochart-testing.yaml:1]: Was this file intended to be included? Looks like it's for testing unless I'm missing something.

Comment (mrballcb) [reply monochart/nginx-config-patch-monochart-testing.yaml:1]: Maybe move it into a `patches/` subdirectory would make its intent more clear?

**Review by mrballcb — APPROVED** [review]

File: monochart/nginx-config-patch-monochart-testing.yaml (hunk 1 above)
Comment (jgonzalezspoton) [reply monochart/nginx-config-patch-monochart-testing.yaml:1]: nope, not supposed to be included, sorry thought I got them all 😂

Comment (jgonzalezspoton) [reply monochart/nginx-config-patch-monochart-testing.yaml]: removed.
